packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -440,6 +443,19 @@ contract OPStackManager is ISemver, Initializable {
         );
     }
 
+    function encodeDisputeGameFactoryInitializer(
+        bytes4 _selector,
+        DeployInput memory _input,
+        DeployOutput memory _output
+    )
+        internal
+        view
+        virtual
+        returns (bytes memory)
+    {
+        return abi.encodeWithSelector(_selector, _output.disputeGameFactoryProxy, _input.roles.opChainProxyAdminOwner);
```

Comment:
Is this the right data to initialize? It looks like the signature is just `initialize(address _owner)`
```suggestion
        return abi.encodeWithSelector(_selector, _input.roles.opChainProxyAdminOwner);
```

Reply:
Good catch 
a59afcb504e0